auto_backup/models/db_backup.py

```diff
@@ -252,7 +253,8 @@ def schedule_backup(self):
                     # Close the SFTP session.
                     sftp.close()
                 except Exception as e:
-                    _logger.debug('Exception! We couldn\'t back up to the FTP server..')
+                    sftp.close()
```

Comment:
Does it really make sense? I assume the `Exception` kills off anything?

Reply:
Apparently it doesn't kill the SFTP connection; we had an issue with accumulating SFTP processes because of this (and the actual bug which caused the removal of old files to fail.)

Reply:
Oh interesting, I did not expect that actually. Then let us add it.